pallets/compliance-manager/src/lib.rs

```diff
@@ -301,26 +355,42 @@ decl_module! {
         #[weight = T::DbWeight::get().reads_writes(1, 1) + 600_000_000 + 1_000_000 * u64::try_from(max(sender_conditions.len(), receiver_conditions.len())).unwrap_or_default()]
```

Comment:
Another component of writing benchmarks is adding the weightinfo trait so that the results from the benchmarks can be used.